go.sum

```diff
@@ -993,6 +981,10 @@ github.com/rancher/wrangler v0.7.3-0.20210331224822-5bd357588083/go.mod h1:goezj
 github.com/rancher/wrangler-api v0.6.1-0.20200427172631-a7c2f09b783e/go.mod h1:2lcWR98q8HU3U4mVETnXc8quNG0uXxrt8vKd6cAa/30=
 github.com/rcrowley/go-metrics v0.0.0-20181016184325-3113b8401b8a/go.mod h1:bCqnVzQkZxMG4s8nGwiZ5l3QUCyqpo9Y+/ZMZ9VjZe4=
 github.com/remyoudompheng/bigfft v0.0.0-20170806203942-52369c62f446/go.mod h1:uYEyJGbgTkfkS4+E/PavXkNJcbFIpEtjt2B0KDQ5+9M=
+github.com/rmweir/lasso v0.0.0-20220421234225-bc78198908bd h1:2hBdsq47jZQrv/80wIoqKcwxcQjDA1Jm7HT2UHklIkE=
+github.com/rmweir/lasso v0.0.0-20220421234225-bc78198908bd/go.mod h1:9qZd/S8DqWzfKtjKGgSoHqGEByYmUE3qRaBaaAHwfEM=
+github.com/rmweir/norman v0.0.0-20220421230622-c2e07fab8a8f h1:EsvjlZ0sR40w+42vEByacVZOf32TiGki1Z1nNlWP5lo=
+github.com/rmweir/norman v0.0.0-20220421230622-c2e07fab8a8f/go.mod h1:hhnf77V2lmZD7cvUqi4vTBpIs3KpHNL/AmuN0MqEClI=
```

Comment:
Similar comment here about the testing libs - leaving this comment to make sure that we get the "testing dependency" stuff removed before we merge this in.